runtime/src/main/java/org/corfudb/runtime/clients/NettyClientRouter.java

```diff
@@ -52,6 +56,17 @@
         implements IClientRouter {
 
     /**
+     * Metrics: meter (counter), histogram
+     */
+    private static Gauge<Integer> gaugeConnected;
+    private static Timer timerConnect;
+    private static Timer timerSyncOp;
+    private static Counter counterConnectFailed;
+    private static Counter counterSendDisconnected;
+    private static Counter counterSendTimeout;
+    private static Counter counterAsyncOpSent;
```

Comment:
Use of `static` here is an error, will fix.  Use of static elsewhere was debated this afternoon, with a not-quite-agreed "static is probably alright" (except for the error here).  I'll go with keeping non-error static uses unless subsequent debate tips back against it.